docs/getting_started/known_issues.md

```diff
@@ -18,13 +18,20 @@ lastupdated: "2021-02-20"
 {: #knownissues}
 
 These are known issues and limitations for {{site.data.keyword.edge_notm}} ({{site.data.keyword.ieam}}) {{site.data.keyword.version}}.
+
+For a full list of open issues for the {{site.data.keyword.ieam}} OpenSource layer, review the GitHub issues in each of the [Open Horizon repositories](https://github.com/open-horizon/).
+
 {:shortdesc}
 
-## Known issues
+## Known issues for {{site.data.keyword.ieam}} {{site.data.keyword.version}}
+
+These are known issues and limitations for {{site.data.keyword.ieam}}.
 
 * [Private image repositories](../developing/container_registry.md) are not supported for service creation in the management console.
 
-* The **-f &lt;directory&gt;** flag of **edgeNodeFiles.sh** does not have the intended effect. Instead, the files are collected in the current directory. For details see [issue 2187](https://github.com/open-horizon/anax/issues/2187). The work around is to run the command like this:
+* {{site.data.keyword.ieam}} does not perform a malware or virus scan on data that is uploaded to the model management system (MMS). For more information about MMS security, see [Security and privacy](../OH/docs/user_management/security_privacy.md#malware).
```

Comment:
The URL path here looks wrong

Reply:
I fixed the link in this commit https://github.com/open-horizon/open-horizon.github.io/pull/220/commits/69afef64f5f1e498f4f331227fc10cd327d472b8